cibuild.cmd

```diff
@@ -1,6 +1,14 @@
 call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" x86
 msbuild /v:m /m BuildAndTest.proj /p:CIBuild=true
+if ERRORLEVEL 1 (
+    echo Build failed
+    exit /b 1
+)
 
 REM Kill any instances of VBCSCompiler.exe to release locked files;
 REM otherwise future CI runs may fail while trying to delete those files.
-taskkill /F /IM vbcscompiler.exe
\ No newline at end of file
+taskkill /F /IM vbcscompiler.exe
```

Comment:
Do we not need to still run this even in the case where the build fails?  In the case of failure, will all vbcscompiler instances be guaranteed to be gone?